docs/conf.py

```diff
@@ -89,7 +89,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+# html_static_path = ['_static']
```

Comment:
Does this line also generate warning?

Reply:
Hi @mkrainiuk , yes, I kept getting build warnings until I commented that line out per Todd R's recommendation.

Reply:
Hi @mkrainiuk , my next commit will address this. I have uncommented line 92 and added a `.../docs/_static` directory with some style/formatting files in it. The docs build without warnings/errors now.

Reply:
@mkrainiuk , Note that with this new commit, I changed the theme in conf.py to sphinx_book_theme. I'm not sure if you have any CI dependencies where that change should be reflected (e.g. docker install manifest, etc.), so I wanted to call that out specifically.